apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerTransactionOptions.cs

```diff
@@ -66,4 +68,22 @@ public TimeSpan? MaxCommitDelay
     }
 
     internal Duration MaxCommitDelayDuration => MaxCommitDelay is null ? null : Duration.FromTimeSpan(MaxCommitDelay.Value);
+
+    // Note: We use seconds here to follow the convention set by DbCommand.CommandTimeout.
+    /// <summary>
+    /// The wait time, in seconds, before terminating the attempt to <see cref="SpannerTransaction.Commit()"/>
+    /// or <see cref="SpannerTransaction.Rollback"/> and generating an error. May be null, in which case, a transaction
+    /// using these options will default to the commit timeout defined on its connection string.
+    /// A value of '0' normally indicates that no timeout should be used (wait an infinite amount of time).
```

Comment:
Presumably the implementation for these semantics lives somewhere else that isn't affected by this PR?